src/containers/NdlaFilm/components/NdlaFilmArticle.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree. *
+ */
+
+import { useField, useFormikContext } from 'formik';
+import { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+import { IconButtonV2 } from '@ndla/button';
+import { FieldHeader } from '@ndla/forms';
+import { colors, spacing } from '@ndla/core';
+import { DeleteForever } from '@ndla/icons/editor';
+import Tooltip from '@ndla/tooltip';
+import { IArticleSummaryV2 } from '@ndla/types-backend/article-api';
+import { IMultiSearchSummary } from '@ndla/types-backend/search-api';
+import DropdownSearch from '../../NdlaFilm/components/DropdownSearch';
+import { getUrnFromId, getIdFromUrn } from '../../../util/ndlaFilmHelpers';
+import { searchArticles } from '../../../modules/article/articleApi';
+
+interface Props {
+  fieldName: string;
+  onUpdateArticle: Function;
+}
+
+const ArticleElement = styled.div`
+  align-items: center;
+  background: ${colors.brand.greyLighter};
+  display: flex;
+  justify-content: space-between;
+  margin: ${spacing.normal} 0;
+  padding: ${spacing.small};
+`;
+
+const NdlaFilmArticle = ({ fieldName, onUpdateArticle }: Props) => {
+  const { t } = useTranslation();
+  const form = useFormikContext();
+  const [field] = useField<string>(fieldName);
+  const [selectedArticle, setSelectedArticle] = useState<null | IArticleSummaryV2>(null);
+
+  const fetchArticle = useCallback(async () => {
+    if (field.value) {
+      const article = await searchArticles({ ids: `${getIdFromUrn(field.value)}` });
+      setSelectedArticle(article.results[0]);
+    } else {
+      setSelectedArticle(null);
+    }
+  }, [field.value]);
+
+  useEffect(() => {
+    fetchArticle();
+  }, [field.value, fetchArticle]);
+
+  return (
+    <>
+      <FieldHeader
+        title={t('ndlaFilm.editor.moreInfoTitle')}
+        subTitle={t('ndlaFilm.editor.moreInfoSubTitle')}
+      />
+      {selectedArticle && (
+        <ArticleElement>
+          <Link
+            to={`/subject-matter/learning-resource/${selectedArticle.id}/edit/${selectedArticle.title.language}`}
+          >
+            {selectedArticle.title.title}
+          </Link>
+          <Tooltip tooltip={t('ndlaFilm.editor.removeArticleFromMoreInformation')}>
```

Comment:
Foretrekk title på knappen fremfor tooltip.

Reply:
Valgte Tooltip her fordi den alt var i bruk lenger ned (på slideshow- og grupperinger-listene) og det er marginal nok forskjell på dem til at det ser litt dumt ut (title-attributtet bruker <title>, som jeg trur er nettleser-native, og ikke mulig å style for å få lik).
Alternativt, hvis du vil, kan jeg gjøre om ElementListItem, slik at den bruker samme, men da blir denne hakket større, sida jeg da mener vi bør også oppdatere de andre sidene som ElementListItem er brukt på, for å ikke forskyve problemet.

Reply:
Tenker det er lurt å få i første omgang denne sida til å være konsistent. Om det betyr å endre andre plasser der ElementListItem brukes så får du heller gjøre det.